.circleci/config.yml

```diff
@@ -41,12 +41,12 @@ jobs:
       - run:
           name: "Running unit tests"
           command: npm run ganache >/dev/null 2>&1 & npm run test
-      - run:
-          name: "Running coverage"
-          command: npm run test:coverage
       - run:
           name: "Testing deployment scripts"
           command: npm run ganache >/dev/null 2>&1 & npm run test:deployment
+      - run:
+          name: "Running coverage"
+          command: npm run test:coverage
```

Comment:
Why has the order here changed?

Reply:
Because coverage is the slowest task to run and I find it convenient to be able to identify an issue in the deployment script earlier.